AUTHORS

```diff
@@ -543,3 +540,58 @@ Dennis Meckel <meckel@datenschuppen.de>
 Harshil Meena <harshil.7535@gmail.com>
 Micky <mickydroch@gmail.com>
 arghdos <arghdos@gmail.com>
+Michele Zaffalon <michele.zaffalon@gmail.com>
+Martha Giannoudovardi <maapxa@gmail.com>
+devangkulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in>
+steph papanik <spapanik21@gmail.com>
+msdousti <msdousti@gmail.com>
+Arif Ahmed <arif.ahmed.5.10.1995@gmail.com>
+Abdullah Javed Nesar <abduljaved1994@gmail.com>
+Lakshya Agrawal <zeeshan.lakshya@gmail.com>
+shruti <shrutishrm512@gmail.com>
+ArifAhmed1995 <arif.ahmed.5.10.1995@gmail.com>
+rohitrango <rohit.rango@gmail.com>
+Hong Xu <hong@topbug.net>
+Ivan Petuhov <ivan@ostrovok.ru>
+Alsheh <alsheh@rpi.edu>
+Marcel Stimberg <marcel.stimberg@ens.fr>
+Alexey Pakhocmhik <cool.Bakov@yandex.ru>
+Tommy Olofsson <tommy.olofsson.90@gmail.com>
+Zulfikar <zulfikar97@gmail.com>
+bsdz <bsdz@users.noreply.github.com>
+Danny Hermes <daniel.j.hermes@gmail.com>
+psqq <pestov-sa@yandex.ru>
+Amit Kumar <aktech@users.noreply.github.com>
+Mohit Chandra <mohit.chandra@research.iiit.ac.in>
+karthikchintapalli <karthik.chintapalli@gmail.com>
+Marcin Briański <marcin.brianski@student.uj.edu.pl>
+andreo <andrey.torba@gmail.com>
+Flamy Owl <flamyowl@protonmail.ch>
+Yicong Guo <guoyicong100@gmail.com>
+mamakancha <varun.garg03@gmail.com>
```

Comment:
Okay. I missed that instruction. I checked `doc/src/aboutus.rst` , [here](https://github.com/sympy/sympy/blob/master/doc/src/aboutus.rst)I do not see anything section about Authors. What changes are expected there? A new section?